src/nationalcareers_toolkit/assets/src/frontend/js/LTCESurvey.js

```diff
@@ -1,33 +1,42 @@
-if((localStorage.getItem('Survey') && document.getElementById('popupsurvey')!= null)){
-    document.getElementById('popupsurvey').style.display = 'none'}
-    else if (document.getElementById('popupsurvey')!= null){document.getElementById('popupsurvey').style.display = 'block'};
+function setSurveyCookie(isTrue = true) {
+    var date = new Date();
+    //date.setTime(date.getTime() + (28 * 24 * 60 * 60 * 1000)); // This is for the actual expiry outside for PROD (28 days)
```

Comment:
Thoughts on this line @PurpleDuckMedia? We could leave it as-is for SIT testing, then swap for PP and Prod?

Reply:
Let's review this later post sit as discussed on call.